src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -259,8 +259,9 @@ private static unsafe int GetClassFactoryForTypeInternal(ComActivationContextInt
         /// Internal entry point for registering a managed COM server API from native code
         /// </summary>
         /// <param name="pCxtInt">Pointer to a <see cref="ComActivationContextInternal"/> instance</param>
-        [RequiresUnreferencedCode("Built-in COM support is not trim compatible", Url = "https://aka.ms/dotnet-illink/com")]
         [UnmanagedCallersOnly]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "The same feature switch applies to GetClassFactoryForTypeInternal and this function. We rely on the warning from GetClassFactoryForTypeInternal.")]
```

Comment:
FYI @jkurdek

We will have to change this - either in this PR or later (doesn't matter much). @jkurdek is working on an experimental feature in linker to detect unnecessary suppressions. That will have problems around feature switches, since in default setup the linker doesn't see the code which the suppression applies to.

In general, it's better to extract the one dangerous method we're calling into a local function and add the suppression on the local function:
* Make it clear which method the suppression is about (we've found lot of places in runtime where it's absolutely not clear)
* If later on somebody modifies the code and it would call into a new method which produces the same warning, it would not warn and it could produce a trimming break.
* And it allows the linker to figure out unnecessary suppressions :-)